Source/Core/DiscIO/Enums.cpp

```diff
@@ -182,6 +182,8 @@ std::string GetSysMenuVersionString(u16 title_version)
   case Region::NTSC_K:
     region_letter = "K";
     break;
+  default:
```

Comment:
Better to add a case for UNKNOWN_REGION than to use a default isnt it ? The whole purpose of the warnings is to alert people to missing handling.